quickstarts/dbmarlin/config.yml

```diff
@@ -0,0 +1,60 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: dbmarlin
+
+# Displayed in the UI (required)
+title: DBmarlin
+
+# Long-form description of the quickstart (required)
+description: |
+  The DBmarlin quickstart allows you to get visibility into the performance and availability of your database services from within New Relic.
+
+  To get started with DBmarlin, visit the [DBmarlin introduction page](https://docs.dbmarlin.com/docs/DBmarlin/introduction).
+
+  # About DBmarlin
+  DBmarlin monitors database performance, tracks changes and helps your databases run fast.
+
+  It provides an easy-to-use and consistent way to monitor CockroachDB, DB2, MariaDB, MySQL, Oracle, PostgreSQL and SQL Server, self-hosted or in the cloud.
+
+  It captures SQL statement text and wait events so you can see exactly where the time is spent executing SQL within your database.
+
+  It auto-detects changes to schema objects, database parameters and execution plans so you can see their impact on performance.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  View database performance from within New Relic with this integration with DBmarlin.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - DBmarlin
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - database
+  - performance
+  - availability
+  - cockroach
+  - db2
+  - mysql
+  - maria
+  - oracle
+  - postgres
+  - sql
+  - newrelic partner
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - third-party-dbmarlin
+
+# Documentation references
+documentation:
+  - name: Installation docs
+    url: https://docs.dbmarlin.com/docs/integrations/newrelic
```

Comment:
This link is currently 404ing for me, does it need to be made public?

Reply:
@aswanson-nr  I don't think the docs are live yet, which is why the links aren't working. Should we wait on this PR until they are? Also, do you know why this is failing? I thought Icons were no longer used in quickstarts.

Reply:
https://docs.dbmarlin.com/docs/integrations/newrelic is not live yet - it is still in staging. I hope to get it live in the next few days.

Reply:
This will immediately be published after merging, so we should hold off on this PR until that documentation is live.

@jcountsNR, it looks like I missed that the dashboards are under the `quickstarts/dbmarlin` directory. Dashboards have been moved to their own top level directory to allow them to be shared between quickstarts. 
We'll just need to get this updated by moving the dashboard over to the `dashboards/` top level directory, then updating the `config.yml` to point to that dashboard. Full docs are available [here](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#quickstart-definitions)

Reply:
Please let me know if either of your run into any issues!

Reply:
https://docs.dbmarlin.com/docs/integrations/newrelic is now live.

Reply:
@aswanson-nr  ah you are right, I missed that one as well. @martinpinner  can you also move the dashboard to the correct directory? The docs are linked above.